modules/persistent-storage-csi-efs-static-pv.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+
+[id="efs-create-static-pv_{context}"]
+= Creating static PVs with AWS EFS
+
+It is possible to use an AWS EFS volume as a single PV without any dynamic provisioning. The whole volume is mounted to pods.
+
+.Procedure
+
+. Create AN EFS volume in AWS. Note the Security Group ID. This is needed later when creating the PV.
```

Comment:
The security group is not mentioned anywhere else. IMO it's worth creating a special chapter that would be a prerequisite for both static and dynamic provisioning.

* Cluster admin must create EFS volume(s).
* They must use the right VPC for the EFS volumes to make sure OCP has access to the volume.
* They must open a firewall port for NFS in the default AWS security group. Maybe even with some screenshots where to find the security group ID and how to add NFS port there. This will be the single most common problems that users will encounter. There should be a troubleshooting chapter that would point it out *again* - if they get timeouts, they should check their networking and firewall. OCP nodes must have access to NFS port of the EFS volumes. This is how it looks like when they don't:

```
$ oc describe pod
...
  Type     Reason       Age    From               Message
  ----     ------       ----   ----               -------
  Normal   Scheduled    2m13s  default-scheduler  Successfully assigned default/efs-app to ip-10-0-135-94.ec2.internal
  Warning  FailedMount  13s    kubelet            MountVolume.SetUp failed for volume "pvc-d7c097e6-67ec-4fae-b968-7e7056796449" : rpc error: code = DeadlineExceeded desc = context deadline exceeded
  Warning  FailedMount  10s    kubelet            Unable to attach or mount volumes: unmounted volumes=[persistent-storage], unattached volumes=[persistent-storage kube-api-access-9j477]: timed out waiting for the condition
```